src/components/Comment.jsx

```diff
@@ -0,0 +1,18 @@
+import { Card } from "react-bootstrap";
+import { ListGroup } from "react-bootstrap";
+
+export const Comment = ({ comment }) => {
+  return (
+    <Card>
+      <ListGroup className="list-group-flush">
+        <ListGroup.Item>{comment.author}</ListGroup.Item>
+        <ListGroup.Item> {comment.created_at}</ListGroup.Item>
```

Comment:
2020-10-17T23:05:00.000Z is not human readable. Find a way to format the date into something more user friendly.